frontend/server/src/DAO/RunsGroups.php

```diff
@@ -19,32 +19,25 @@ final public static function getProblemsetRunsGroups(
         int $problemsetId
     ): array {
         $sql = "SELECT
-                    IFNULL(AVG(mspg.score), 0.0) AS score,
-                    AVG(mspg.score) * pp.points AS contest_score,
-                    r.penalty,
-                    s.problem_id,
-                    s.identity_id,
-                    s.`type`,
-                    s.`time`,
-                    s.`submit_delay`,
-                    s.`guid`
+                    IFNULL(SUM(mspg.score), 0.0) AS score,
+                    SUM(mspg.score) * pp.points AS contest_score,
+                    0 AS penalty,
```

Comment:
```suggestion
                    SUM(mspg.penalty) AS penalty,
```

o quizás MAX? supongo que depende del concurso, hay que proveer manera de cambiarlo.

Reply:
Listo.